CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+1.7.0
+=====
+* [TG-4312] Expose private overload of `setField`
```

Comment:
Is this really all we've done this entire sprint?

Reply:
No, its all I've done to deeptest utils.